src/server/routes/page.js

```diff
@@ -393,9 +395,11 @@ module.exports = function(crowi, app) {
       page = await page.seen(req.user);
     }
 
+    const compareRevisionIds = compare ? compare.split('...') : [];
+
     // populate
     page = await page.populateDataToShowRevision();
-    addRenderVarsForPage(renderVars, page);
+    addRenderVarsForPage(renderVars, page, compareRevisionIds);
```

Comment:
Don't change addRenderVarsForPage specs. It will be removed when GROWI upgrade to use next.js.